app/controllers/items_controller.rb

```diff
@@ -24,31 +24,16 @@ def create
 
   def update
     item.update!(item_params)
-    respond_to do |format|
-      format.json do
-        render json: item.as_json(only: [:id, :desc, :completed])
-      end
-
-      format.html do
-        flash[:notice] = "Item has been updated"
-        redirect_to workspace
-      end
-    end
+
+    flash.now[:notice] = "Item has been updated"
+    render partial: "update", locals: {item}, content_type: "text/vnd.turbo-stream.html"
   end
 
   def destroy
     item.destroy!
 
-    respond_to do |format|
-      format.json do
-        render json: {deletedId: item.id}
-      end
-
-      format.html do
-        flash[:notice] = "Item has been deleted"
-        redirect_to workspace
-      end
-    end
+    flash.now[:notice] = "Item has been deleted"
+    render partial: "update", locals: {item}, content_type: "text/vnd.turbo-stream.html"
```

Comment:
Does `locals: {item}` expand to `locals: {item: item}` somehow or is this a JavaScript-brain typo?

Reply:
That's right, @javan. In ES6, there's a shorthand version for properties with a value coming from a variable with the same name. So, having `{ item }` will is the same as writing `{ x: x }`

Reply:
This is a Ruby file though

Reply:
/facepalm. You are right.

Reply:
> Does locals: {item} expand to locals: {item: item} somehow

Yes, via [Ruby Next](https://github.com/ruby-next/ruby-next#proposed-and-edge-features) 🙂

It's a proposed feature: https://bugs.ruby-lang.org/issues/15236

Reply:
🤗